js/PatchSelectWidget.js

```diff
@@ -12,22 +12,23 @@ $( function () {
 			pd.patchIndex[ patchKey ].push( wiki );
 		}
 	}
-} );
+}() );
 
 window.PatchSelectWidget = function PatchSelectWidget( config ) {
 	var widget = this;
 
-	// Parent constructor
-	window.PatchSelectWidget.super.call( this, $.extend( {
-		allowArbitrary: true,
-		allowDisplayInvalidTags: true
-	}, config ) );
-
 	this.$formInput = $( '<input>' ).attr( {
```

Comment:
$formInput needs to be available immediately if the parent constructor changes the value